landing/src/components/modules/top-navigation.tsx

```
@@ -1,50 +1,58 @@
+
+        <Button onClick={async () => await mint_test_lords({ signer: account, lords_address: lordsAddress })}>
+          Mint Test Lords
+        </Button>
+
+        <>
+          {connectors.map((connector, index) => (
+            <Button size={"default"} key={index} onClick={() => connect({ connector })} variant="outline">
+              <img className="w-5" src={typeof connector.icon === "string" ? connector.icon : connector.icon.dark} />{" "}
+            </Button>
+          ))}
+        </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)